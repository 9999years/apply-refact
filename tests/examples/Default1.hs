yes = foo . bar . concat . map f . baz . bar