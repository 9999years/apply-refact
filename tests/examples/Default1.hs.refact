[("tests/examples/Default1.hs:1:19: Error: Use concatMap\nFound:\n  concat . map f . baz . bar\nWhy not:\n  concatMap f . baz . bar\n",[Replace {rtype = Expr, pos = SrcSpan {startLine = 1, startCol = 19, endLine = 1, endCol = 45}, subts = [("f",SrcSpan {startLine = 1, startCol = 32, endLine = 1, endCol = 33}),("x",SrcSpan {startLine = 1, startCol = 36, endLine = 1, endCol = 45})], orig = "concatMap f . x"}])]
