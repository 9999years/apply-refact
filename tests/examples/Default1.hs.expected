yes = foo . bar . concatMap f . baz . bar
